Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2921,6 +2920,7 @@ - (void)yapDatabaseModified:(NSNotification *)notification
                         rowChange.newIndexPath);
                     [self.collectionView deleteItemsAtIndexPaths:@[ rowChange.indexPath ]];
                     [self.collectionView insertItemsAtIndexPaths:@[ rowChange.newIndexPath ]];
+                    [rowsThatChangedSize removeObject:@(rowChange.newIndexPath.row)];
```

Comment:
We don't want to reload a row that we just moved.

Reply:
Maybe leave this GH comment in the code. 

This is a perf win, but isn't intended to address correctness, right?

Reply:
This is all about correctness - `performBatchUpdates` will throw an exception if you update a row that you also insert.